misc_scripts/change_toolchain_version.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+"""
+change_toolchain_version: change the toolchain version in the GitHub Actions
+workflow configurations.
+"""
+
+# isort: STDLIB
+import argparse
+
+KEY_LSRT = r"# LOWEST SUPPORTED RUST TOOLCHAIN"
+KEY_CDRT = r"# CURRENT DEVELOPMENT RUST TOOLCHAIN"
+TOOLCHAIN_CHOICES = ["lowest", "current"]
+
+
+def search_file(args, search_key, old_verstring, new_verstring, filename):
```

Comment:
Instead of passing ```args```, the whole namespace, pass only what's needed. It's a good time to document the parameters as well.

Reply:
Resolved.